File: EvenOrOdd/EvenOrOdd.c
#include <stdio.h>

int main(void)
{
    // prompt for a number
    printf("Enter a number ");

    int num;
    scanf("%d",&num);
    
    // check if the number is even and if it is say so, if not say it is not
    if ((num%2)==0)
    {
        printf("This number is even.\n");
    }
    else
    {
        printf("This number is odd.\n");
    }

    return 0;
}

File: Basic/EvenOrOdd/EvenOrOdd.java
import java.util.Scanner;

public class EvenOrOdd {

    public static void main(String[] args) {
        
        // scanner object
        Scanner scan = new Scanner(System.in);
        
        //prompt for a number
        System.out.print("Enter a number ");
        int num = scan.nextInt();
        
        //check if the number is even or odd and print out the result
        if (num % 2 ==0)
        {
            System.out.print("The number is even.\n");
        }
        else
        {
            System.out.print("The number is odd.\n");
        }
    }
    
}
